.travis.yml

```diff
@@ -42,31 +42,31 @@ jobs:
       go: "1.10.x"
     - stage: Build and Verify
       script:
-        - sudo "PATH=$PATH" make testunit
-        - make
+        - sudo "PATH=$PATH" make BUILDTAGS='btrfs_noversion libdm_no_deferred_remove seccomp selinux containers_image_ostree_stub libpod' testunit
+        - make BUILDTAGS='btrfs_noversion libdm_no_deferred_remove seccomp selinux containers_image_ostree_stub libpod'
       go: "1.9.x"
     - script:
-        - sudo "PATH=$PATH" make testunit
-        - make
+        - sudo "PATH=$PATH" make BUILDTAGS='btrfs_noversion libdm_no_deferred_remove seccomp selinux containers_image_ostree_stub libpod' testunit
+        - make BUILDTAGS='btrfs_noversion libdm_no_deferred_remove seccomp selinux containers_image_ostree_stub libpod'
       go: "1.10.x"
     - script:
         - make local-cross
       go: "1.10.x"
       env: CROSS_PLATFORM=true
     - script:
         - sudo "PATH=$PATH" make testunit
-        - make
+        - make 
```

Comment:
You're introducing trailing whitespace here and down on line 62.

Reply:
fixed